t/03scalar.t

```diff
@@ -7,20 +7,15 @@
 # Change 1..1 below to 1..last_test_to_print .
 # (It may become useful if the test is moved to ./t subdirectory.)
 
-our $can_cow;
-BEGIN {
-    # skip the tests when failing to load B::COW
-    $can_cow = eval { require B::COW; B::COW::can_cow() };
-}
-
 use B q{svref_2object};
+use B::COW;
```

Comment:
By reverting this protection would not this break like #36 report?
I would prefer preserving that protection here.

Reply:
Also note that we have a workflow confirming you can install Clone, without `B::COW`
The idea is we do not want to add some extra blockers to install Clone, when that package is only used for testing.

Reply:
The problem in #36 only happened because the prereqs weren't recognized. With this change, the prereqs will be properly installed.

I don't see the advantage of allowing the tests to pass in the case where the prereqs haven't been satisfied.

Reply:
Yes, I hadn't noticed that when creating this patch.